src/css/custom.css

```diff
@@ -50,4 +50,15 @@ footer {
   --ifm-color-primary-light: #78a9fd;
   --ifm-color-primary-lighter: #88b4fe;
   --ifm-color-primary-lightest: #bbd4fe;
-}
\ No newline at end of file
+}
+
+@media (min-width: 997px) {
+  .navbar-title {
+    position: absolute;
+    margin-top: 20px;
+    margin-left: 283px;
```

Comment:
if you change margin left to 270px, itll line up with the end of the sidebar. Personally I think it looks nicer lined up 
![image](https://github.com/immersve/immersve-docs/assets/93644855/308aa713-85d4-4013-a233-450924fb6fc7)

Reply:
Hmmm. That doesn't do it for me. Any other reviewers have feelings on this?

Reply:
I also think it looks better with a bit of a gap. Looks a bit strange lined up with the border like this

Reply:
fair enough! personal preference